### DISCUSSION TITLE
Refactoring of the front end. Make request...

### DISCUSSION THREAD

**Discussion by prpanto** [Kovah/LinkAce#608] [General]
Hello guys,
I try to make a plan to refactoring the front end of the linkace. I will finish another refactoring of other lib and I will start here.
I want a free self hosting raindrop version of this project,
My options about design is:
https://dribbble.com/shots/6970082-Bookmarking-App
https://dribbble.com/shots/3339378-Chrome-Bookmarks
something like that,
What do think? Do you have something better to suggest?

**Comment by Kovah** [comment]
Hi! LinkAce is an open source application, so anybody can use it and modify it to their liking.

I personally want the frontend to stay as it is. You might create your own version. However, please keep in mind that LinkAce has a strict license and modifications have to be published and made open source too. If you plan to create an online service with the app, I suggest you consult a lawyer first.

**Reply by prpanto** [reply]
hey @Kovah thanks for the reply. I like your project and I dont want to publish as a service. But I would like some simple or basic front end changes.
